src/web/badge/src/lib.rs

```diff
@@ -33,95 +26,106 @@ impl Default for BadgeOptions {
     }
 }
 
-
 pub struct Badge {
     options: BadgeOptions,
     font: Font<'static>,
     scale: Scale,
     offset: Point<f32>,
 }
 
-
 impl Badge {
     pub fn new(options: BadgeOptions) -> Result<Badge, String> {
         let collection = FontCollection::from_bytes(FONT_DATA).expect("Failed to parse FONT_DATA");
+
         // this should never fail in practice
-        let font = collection.into_font().map_err(|_| "Failed to load font data".to_owned())?;
+        let font = collection
+            .into_font()
+            .map_err(|_| "Failed to load font data".to_owned())?;
+
         let scale = Scale {
             x: FONT_SIZE,
             y: FONT_SIZE,
         };
+
         let v_metrics = font.v_metrics(scale);
         let offset = point(0.0, v_metrics.ascent);
+
         if options.status.is_empty() || options.subject.is_empty() {
             return Err(String::from("status and subject must not be empty"));
         }
+
         Ok(Badge {
-            options: options,
-            font: font,
-            scale: scale,
-            offset: offset,
+            options,
+            font,
+            scale,
+            offset,
         })
     }
 
-
     pub fn to_svg_data_uri(&self) -> String {
-        format!("data:image/svg+xml;base64,{}",
-            Base64Display::standard(self.to_svg().as_bytes()))
+        format!(
+            "data:image/svg+xml;base64,{}",
+            Base64Display::with_config(
+                self.to_svg().as_bytes(),
+                Config::new(CharacterSet::UrlSafe, true)
```

Comment:
Oh this is a really good change, it would be worth having on its own. Can you  check whether this fixes https://github.com/rust-lang/docs.rs/issues/395 and if so add a test?

You can test manually like so:
```
docker-compose run web build crate zstd 0.5.1+zstd.1.4.4
```
and then navigating to http://localhost:3000/zstd/badge.svg?version=0.5.1+zstd.1.4.4.

(actually I'll just do this myself, give me a second)

Reply:
Update: this does not fix #395. No need to make it a separate PR.